Makefile

```diff
@@ -9,16 +9,20 @@ LDFLAGS = -lpthread -lboost_system
 PARSER_PATH = ./nginx-configparser/
 GTEST_DIR = nginx-configparser/googletest/googletest
 GTEST_FLAGS = -isystem ${GTEST_DIR}/include
+TEST_COV = --coverage # --coverage is a synonym for-fprofile-arcs, -ftest-coverage(compiling) and-lgcov(linking).
 
 # TODO: Split out dependency handling so that we can avoid rebuilding
-.PHONY: $(TARGET) $(TESTS) all test clean
+.PHONY: $(TARGET) $(TESTS) all test integration_test clean
 
 TARGET = lightning
 TESTS = server_config_test
 SRC = $(PARSER_PATH)config_parser.cc lightning_main.cc lightning_server.cc server_config.cc request_handlers.cc
 
 all: $(TARGET)
 
+test: LDFLAGS += $(TEST_COV)
```

Comment:
Could you explain what's happening here? On the test target, `LDFLAGS` is modified? I though Make only supported listing dependencies here.